go/vt/vtgate/vtgate.go

```diff
@@ -373,7 +387,6 @@ func (vtg *VTGate) Prepare(ctx context.Context, session *vtgatepb.Session, sql s
 
 	fld, err = vtg.executor.Prepare(ctx, "Prepare", NewSafeSession(session), sql, bindVariables)
 	if err == nil {
-		vtg.rowsReturned.Add(statsKey, int64(len(fld)))
```

Comment:
`Prepare` is adding the number of `fieldInfo` rows to `RowsReturned`. This seemed wrong so I removed it.